src/cloud/project/project-patch.md

```diff
@@ -11,6 +11,8 @@ functional_areas:
 -  {{ site.data.var.mcp-prod }} delivers required patches with critical fixes
 -  Quality Patches deliver optional, low-impact quality fixes as [individual patches]({{ site.baseurl }}/release/policy/#individual-patch), which do not contain backward incompatible changes.
 
+See [patches available in the Quality Patches Tool]({{ site.baseurl }}/quality-patches/tool.html) to review a full list of released patches.
```

Comment:
Actually, the link on line 9 above is redirecting to the link you inserted here. I would suggest you update the Quality Patches Tool link in line 9 and then you don't have to add this line here. And we get rid of a redirect 😄

Reply:
?? 
in Line 9 I see links on repositories 
```
[{{ site.data.var.mcp-prod }}](https://github.com/magento/magento-cloud-patches) and the [Quality Patches Tool](https://github.com/magento/quality-patches) deliver patches to your installed {{site.data.var.ee}} application.
```
Those links are totally different.
It's an internal PR, can we have it merged ASAP?
If anything else is required from me please ping me in a slack @tymchyns 
Thank you

Reply:
It is a different URL, but if you click on it from the topic page it redirects now to the URL you are trying to add.